baselines/jft/al_utils.py

```diff
@@ -122,7 +121,6 @@ def as_dataset(self,
     dataset = tf.data.Dataset.from_generator(
         _subset_generator(
             dataset=dataset,
-            dataset_info=self.base_dataset_builder.info,
```

Comment:
Did you mean to delete this entirely? Looks like `_subset_generator` still requires a `dataset_info` argument.